aptos/contracts/Makefile

```diff
@@ -0,0 +1,14 @@
+.PHONY: artifacts
+artifacts: build
+
+.PHONY: build
```

Comment:
not for this PR: can we somehow add a github CI check for this?

Reply:
yep, I will add Aptos to the Tilt environment in a follow-up PR 😄